templates/usr/local/lib/owncloud-enforce-permissions.j2

```diff
@@ -5,11 +5,10 @@
 
 # {{ ansible_managed }}
 
-[[ "$UID" != 0 ]] && { echo "You must be root"; exit 1; }
-
```

Comment:
Well, that is a valid point of course but just that the execution permissions is not there does not mean that no user will try to run the script :wink: